crates/types/src/types.rs

```diff
@@ -449,3 +449,12 @@ pub struct AccountUpdate {
     /// providing this value sets the epoch at which this account will next owe rent
     pub rent_epoch: Option<Epoch>,
 }
+
+#[derive(Serialize, Debug, Clone)]
+pub struct TokenAccountUpdateParams {
```

Comment:
We have the `TokenAccountUpdate` type in the `surfnet_cheatcodes` file that you can move to here, just like you did with the `AccountUpdate` type